apps/web/playwright/organization/booking-list.e2e.ts

```diff
@@ -162,17 +162,19 @@ test.describe("Bookings list for organizations", () => {
 
       const eventType = await team1owner.getFirstTeamEvent(team1.id);
 
-      await expectPageToBeNotFound({ page, url: `/team/${team1.slug}/${eventType.slug}` });
+      // await expectPageToBeNotFound({ page, url: `/${commonUser.username}/${eventType.slug}` });
 
-      await doOnOrgDomain(
-        {
-          orgSlug: org.slug,
-          page,
-        },
-        async () => {
-          await bookTeamEvent({ page, team: team1, event: eventType });
-        }
-      );
+      // await doOnOrgDomain(
+      //   {
+      //     orgSlug: org.slug,
+      //     page,
+      //   },
+      //   async () => {
+      //     await bookUserEvent({ page, user: commonUser, event: eventType });
```

Comment:
and bookUserEvent() is not working insidw doOnOrgDomain() (also for users in org)
moved it outside, works

Reply:
@hariombalhara @Udit-takkar Any ideas on this one?

Reply:
I just tried uncommenting it and it worked fine for me. make sure you have setup organization locally https://github.com/calcom/cal.com/blob/main/packages/features/ee/organizations/README.md

<img width="967" alt="Screenshot 2024-03-06 at 3 42 50 PM" src="https://github.com/calcom/cal.com/assets/53316345/757ba553-9fa6-4b94-a2c2-bd24ea37a73b">

Reply:
it must be something with my local setup then 👍 
uncommented and pushed